### ISSUE TITLE
3D Tiles refinement broken when skip LODs is disabled

### ISSUE THREAD

**Issue by bagnell** [AnalyticalGraphicsInc/cesium#5814]
```javascript
var viewer = new Cesium.Viewer('cesiumContainer');

var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
    url : 'https://beta.cesium.com/api/assets/1458?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIxYmJiNTAxOC1lOTg5LTQzN2EtODg1OC0zMWJjM2IxNGNlYmMiLCJpZCI6NDQsImFzc2V0cyI6WzE0NThdLCJpYXQiOjE0OTkyNjM4MjB9.1WKijRa-ILkmG6utrhDWX6rDgasjD7dZv-G5ZyCmkKg',
    skipLevelOfDetail : false
}));

tileset.readyPromise.then(function() {
    var boundingSphere = tileset.boundingSphere;
    viewer.camera.viewBoundingSphere(boundingSphere, new Cesium.HeadingPitchRange(Cesium.Math.PI_OVER_TWO, -0.5, boundingSphere.radius * 0.1));
    viewer.camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
}).otherwise(function(error) {
    throw(error);
});
```
![image](https://user-images.githubusercontent.com/1494815/30129707-d42d77e6-9314-11e7-8c28-23f3ab80fbc7.png)

**Comment by bagnell** [comment]
If you try the example, the tileset won't refine until the camera is moved.

**Comment by pjcozzi** [comment]
@lilleyse can you please look at this?  Perhaps this was broken with the recent fix in 1.37?

Please make sure there are enough unit tests and hooks into tile traversal to reasonably test.
